src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/user';
+
+type Props = {
```

Comment:
if this type is needed for export, then it’s better to let us know whose props they are
like this `TodoFormProps`